catalyst/utils/tests/test_config.py

```diff
@@ -17,30 +17,29 @@ def test_parse_config_args():
 
     parser = argparse.ArgumentParser()
     parser.add_argument("--command")
+    parser.add_argument("--logdir", type=str, default=None)
+    parser.add_argument(
+        "--autoresume",
+        type=str,
+        required=False,
+        choices=["best", "last"],
+        default=None,
+    )
 
     args, uargs = parser.parse_known_args(
-        [
-            "--command",
-            "run",
-            "--path=test.yml:str",
-            "--stages/zero=cero:str",
-            "-C=like:str",
-        ]
+        ["--command", "run", "--logdir", "logdir", "--autoresume", "last"]
     )
 
     configuration, args = utils.parse_config_args(
         config=configuration, args=args, unknown_args=uargs
     )
 
     assert args.command == "run"
-    assert args.path == "test.yml"
     assert configuration.get("stages") is not None
-    assert "zero" in configuration["stages"]
-    assert configuration["stages"]["zero"] == "cero"
-    assert configuration.get("args") is not None
-    assert configuration["args"]["path"] == "test.yml"
-    assert configuration["args"]["C"] == "like"
-    assert configuration["args"]["command"] == "run"
+    assert "logdir" in configuration["args"]
+    assert configuration["args"]["logdir"] == "logdir"
+    assert "autoresume" in configuration["args"]
+    assert configuration["args"]["autoresume"] == "last"
```

Comment:
If your config is now OmegaConf config, you can switch to a nicer attribute style access, e.g `configuration.args.logdir`.

Reply:
Ok. But this feature will be in next iteration I think.